### DISCUSSION TITLE
Integration of Calender Events

### DISCUSSION THREAD

**Discussion by tfischer90** [Blackymas/NSPanel_HA_Blueprint#1536] [Ideas]
Hi, I would like to use the panel to get Calender Events like bring out the trash today.  Would that be possible to integrate? Or do you have an idea of a workaround? I am still new to the HA enviroment.

Thanks a lot in advance.

**Comment by quenthal** [comment]
For trash day you could make template binary sensor or toggle helper with automation to switch it on/off. Then use this as chip with icon you like.

**Comment by edwardtfn** [comment]
- #1731
